repo/packages/M/marathon/300/config.json

```diff
@@ -0,0 +1,571 @@
+          "type" : "string",
+          "default" : "*"
+        },
+        "default-network-name" : {
+          "description" : "Network name, injected into applications' container-mode network{} specs that do not define their own name.",
+          "type" : "string"
+        },
+        "deprecated-features" : {
+          "description" : "A comma-separated list of deprecated features to continue to enable",
+          "type" : "string",
+          "default" : ""
+        },
+        "disable-http" : {
+          "description" : "Disable listening for HTTP requests completely. HTTPS is unaffected.",
+          "type" : "boolean",
+          "default" : false
+        },
+        "draining-seconds" : {
+          "description" : "(Default: 0 seconds) the seconds when marathon will start declining offers before a maintenance window start time. This has no effect if '--disable_maintenance_mode' is specified!",
+          "type" : "integer",
+          "default" : 0
+        },
+        "enable-features" : {
+          "description" : "A comma-separated list of features. Available features are: secrets - Enable support for secrets in Marathon (experimental), external_volumes - Enable external volumes support in Marathon, maintenance_mode - (on by default, has no effect) Decline offers from agents undergoing a maintenance window, vips - Enable networking VIPs UI, gpu_resources - Enable support for GPU in Marathon (experimental), task_killing - Enable the optional TASK_KILLING state, available in Mesos 0.28 and later",
+          "type" : "string",
+          "default" : "vips,task_killing"
+        },
+        "env-vars-prefix" : {
+          "description" : "Prefix to use for environment variables injected automatically into all started tasks.",
+          "type" : "string"
+        },
+        "event-stream-max-outstanding-messages" : {
+          "description" : "The event stream buffers events, that are not already consumed by clients. This number defines the number of events that get buffered on the server side, before messages are dropped.",
+          "type" : "integer",
+          "default" : 1024
+        },
+        "executor" : {
+          "description" : "Executor to use when none is specified. If not defined the Mesos command executor is used by default.",
+          "type" : "string",
+          "default" : "//cmd"
+        },
+        "failover-timeout" : {
+          "description" : "(Default: 1 week) The failover_timeout for mesos in seconds.",
+          "type" : "integer",
+          "default" : 604800
+        },
+        "framework-name" : {
+          "description" : "(Default: service.name) Framework name to register with Mesos.",
+          "type" : "string"
+        },
+        "gpu-scheduling-behavior" : {
+          "description" : "Defines how offered GPU resources should be treated. Possible settings are `undefined`, `restricted` and `unrestricted`",
+          "type" : "string",
+          "default" : "undefined"
```

Comment:
@kensipe Please make this an `enum`: see https://github.com/mesosphere/universe/pull/2017 from @deric 
```
          "enum": [
            "undefined",
            "restricted",
            "unrestricted"
          ]
```